src/AmalgamatorSession.ts

```diff
@@ -675,6 +707,20 @@ export class AmalgamatorSession extends LoggingDebugSession {
             ].customRequest('cdt-gdb-adapter/Memory', args);
             response.body = childResponse.body;
             this.sendResponse(response);
+        } else if (command === 'cdt-amalgamator/resumeAll') {
+            const [, threads] = await this.collectChildTheads();
+            threads.forEach(async (thread) => {
+                const [childIndex, childId] = await this.getThreadInfo(
+                    thread.id
+                );
+                args = { threadId: childId } as DebugProtocol.ContinueArguments;
+                await this.childDaps[childIndex].customRequest(
+                    'cdt-gdb-adapter/resumeAll',
+                    args
+                );
+            });
+            response.body = 'OK';
```

Comment:
AFAIK You don't need to have a body for a response.

Reply:
I updated it